src/main/scripts/festerize/README.md

```diff
@@ -4,11 +4,11 @@ Uploads CSV files to the Fester IIIF manifest service for processing.
 
 Any collection rows found in the CSV are used to create a IIIF collection.
 
-Any work rows are used to expand or revise a previously created IIIF collection (corresponding to the collection that the work is a part of), as well as create a IIIF manifest corresponding to the work.
+Any work rows are used to expand or revise a previously created IIIF collection (corresponding to the collection that the work is a part of), as well as create a IIIF manifest corresponding to the work. A "work" is conceived of as a discrete object that should be access as an individual item. An example of a "work" is a book or a photograph. 
 
-Any page rows are likewise used to expand or revise a previously created IIIF manifest (corresponding to the work that the page is a part of).
+Any page rows found in a CSV are likewise used to expand or revise a previously created IIIF manifest (corresponding to the work that the page is a part of).
 
-After Fester creates or updates any IIIF collections or manifests, it updates and returns the CSV files to the user.
+After Fester creates or updates any IIIF manifests for collections or works, it updates and returns the CSV files to the user.
```

Comment:
We discussed on Slack and decided to keep the original wording, to conform with the IIIF terminology.